### PR TITLE
docs: Add the changelog of 3.26.9

### PR DESCRIPTION
**Pull request by dengfuping**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | `None`          |
| 🇨🇳 Chinese | `None`          |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/changelog-3.26.9/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/changelog-3.26.9/CHANGELOG.zh-CN.md)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ timeline: true
...
 
+## 3.26.9
+
+`2020-02-08`
+
+- 🐞 Fix Badge that count is not align when wrapped by Typography. [#21237](https://github.com/ant-design/ant-design/pull/21237)
+- 🐞 Fix Steps that icon is not align when `size="small"` and `labelPlacement="vertical"`. [#21258](https://github.com/ant-design/ant-design/pull/21258)
+- 🐞 Fix Typography not focus at the end of textarea when `editable` is true. [#21268](https://github.com/ant-design/ant-design/pull/21268)
+- TypeScript
+  - 💄 Improve type definition of validation rule's type for Form. [#21250](https://github.com/ant-design/ant-design/pull/21250) [@hansololai](https://github.com/hansololai) - 🐞 Fix type definition of `event` for Tree. [#21200](https://github.com/ant-design/ant-design/pull/21200) [@Jirka-Lhotka](https://github.com/Jirka-Lhotka)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: 混一行了

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,14 @@ timeline: true
...
 ---
 
+`2020-02-08`
+
+- 🐞 修复 Badge 在 Typography 下数字错位的问题。[#21237](https://github.com/ant-design/ant-design/pull/21237)
+- 🐞 修复 Steps 在 `size="small"` 和 `labelPlacement="vertical"` 时图标没有对齐的问题。[#21258](https://github.com/ant-design/ant-design/pull/21258)
+- 🐞 修复 Typography 在可编辑状态时光标没有在输入框末位的问题。[#21268](https://github.com/ant-design/ant-design/pull/21268)
+- TypeScript
+  - 💄 完善 Form 中校验规则类型的类型定义。[#21250](https://github.com/ant-design/ant-design/pull/21250) [@hansololai](https://github.com/hansololai) - 🐞 修复 Tree 中事件类型定义不正确的问题。[#21200](https://github.com/ant-design/ant-design/pull/21200) [@Jirka-Lhotka](https://github.com/Jirka-Lhotka)
+
```
Comment (chenshuai2144) [inline CHANGELOG.zh-CN.md]: 少了换行符

File: CHANGELOG.en-US.md (hunk 1 above)
Comment (dengfuping) [reply CHANGELOG.en-US.md]: - 奇怪，从语雀上写完 changelog 然后粘贴到 md 文件中，prettier 以后就会出现这种情况。以后还是直接在本地仓库里写吧😂。

Comment (dengfuping) [reply CHANGELOG.en-US.md]: - done

File: CHANGELOG.zh-CN.md (hunk 2 above)
Comment (dengfuping) [reply CHANGELOG.zh-CN.md]: - done.

**Review by afc163 — APPROVED** [review]

**Pull request merged**